ngraph_bridge/enable_variable_ops/ops/ngraph_ops.cc

```diff
@@ -22,6 +22,22 @@ namespace tensorflow {
 
 namespace ngraph_bridge {
 
+// ------------------------------------------------------------------
```

Comment:
Minor: Can you move the registration for Momentum after Gradient -> basically alphabetically :)